Makefile

```diff
@@ -34,61 +41,73 @@ lint:
 test:
 	python -m pytest tests
 
+# docs commands
+
+docs: check-docs
+	open docs/_build/html/index.html
+
+linux-docs: check-docs
+	xdg-open docs/_build/html/index.html
+
 autobuild-docs:
 	sphinx-autobuild --open-browser docs docs/_build/html
 
-build-docs:
-	sphinx-apidoc -o docs/ . setup.py "*conftest*"
-	$(MAKE) -C docs clean
-	$(MAKE) -C docs html
-	$(MAKE) -C docs doctest
-
-build-docs-ci:
-	$(MAKE) -C docs latexpdf
-	$(MAKE) -C docs epub
+# docs helpers
 
 validate-newsfragments:
 	python ./newsfragments/validate_files.py
 	towncrier build --draft --version preview
 
 check-docs: build-docs validate-newsfragments
 
+build-docs:
+	sphinx-apidoc -o docs/ . setup.py "*conftest*"
+	$(MAKE) -C docs clean
+	$(MAKE) -C docs html
+	$(MAKE) -C docs doctest
+
 check-docs-ci: build-docs build-docs-ci validate-newsfragments
 
-docs: check-docs
-	open docs/_build/html/index.html
+build-docs-ci:
+	$(MAKE) -C docs latexpdf
+	$(MAKE) -C docs epub
 
-linux-docs: check-docs
-	xdg-open docs/_build/html/index.html
+# release commands
 
-check-bump:
-ifndef bump
-	$(error bump must be set, typically: major, minor, patch, or devnum)
-endif
+package-test: clean
+	python -m build
+	python scripts/release/test_package.py
 
-notes: check-bump validate-newsfragments
+notes: check-bump
```

Comment:
running `make notes bump=patch` gives me a commit message of `v0.1.1`, so I don't think the eval is doing quite the right thing.